static/js/services/db-sync.js

```diff
@@ -40,6 +45,7 @@ var _ = require('underscore'),
           live: true,
           retry: true,
           timeout: false,
+          heartbeat: 10000,
```

Comment:
No. I think it's clearer to be explicit. I learned a fair bit about heartbeats and timeouts...